src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java

```diff
@@ -300,9 +304,16 @@ public BigDecimal total() {
       this.code = code;
     }
 
+    /**
+     * Convert the given string into an EncounterType.
+     * 
+     * @param value the string to convert.
+     */
     public static EncounterType fromString(String value) {
       if (value == null) {
         return EncounterType.AMBULATORY;
+      } else if (value.equals("super")) {
```

Comment:
Why would Encounter type ever be equal to "super"?

Reply:
At CCDAExporter.java line 61, it creates an encounter with the type "super" which was causing tests to fail unless I put that additional check in.